bin/qgis/smoderp2d-plugin/smoderp_2D_dockwidget.py

```diff
@@ -386,6 +384,49 @@ def OnRunButton(self):
                 "CRITICAL"
             )
 
+    @staticmethod
+    def _layerColorRamp(layer):
+        # get min/max values
+        stats = layer.dataProvider().bandStatistics(1, QgsRasterBandStats.All, layer.extent(), 0)
+
+        # get colour definitions
+        renderer = QgsSingleBandPseudoColorRenderer(layer.dataProvider(), 1)
+        color_ramp = QgsGradientColorRamp(QColor(239, 239, 255), QColor(0,   0, 255))
+        renderer.setClassificationMin(stats.minimumValue)
+        renderer.setClassificationMax(stats.maximumValue)
+        renderer.createShader(color_ramp)
+
+        return renderer
+
+    def computationFinished(self):
+        # clear message bar
+        self.iface.messageBar().clearWidgets
+
+        # show results
+        root = QgsProject.instance().layerTreeRoot()
+        group = root.insertGroup(0, "SMODERP2D")
+
+        outdir = self.main_output_lineEdit.text().strip()
+        first = True
+        for map_path in glob.glob(os.path.join(outdir, '*.asc')):
```

Comment:
I would suggest not using `glob` but holding the list of outputs somewhere and looping through it instead. I understand that this is much more robust and prone to output changes, but can get wrong in the case somebody defines `outdir` as an existing directory already containing some `*.asc` files.

Reply:
@pesekon2 This should not happen. The output directory is cleaned before computation starts. Let's keep it simply for now and use `glob`. Feel free to create separated issue for that.